roofit/roofitcore/inc/RooAbsDataStore.h

```diff
@@ -93,14 +98,18 @@ class RooAbsDataStore : public TNamed, public RooPrintable {
     printStream(defaultPrintStream(),defaultPrintContents(options),defaultPrintStyle(options));
   }
 
-  virtual void printName(std::ostream& os) const ;
-  virtual void printTitle(std::ostream& os) const ;
-  virtual void printClassName(std::ostream& os) const ;
-  virtual void printArgs(std::ostream& os) const ;
-  virtual void printValue(std::ostream& os) const ;
-  void printMultiline(std::ostream& os, Int_t content, Bool_t verbose, TString indent) const ;
-
-  virtual Int_t defaultPrintContents(Option_t* opt) const ;
+  /// Print name of dataset
+  void printName(std::ostream& os) const final { os << GetName(); }
+  /// Print title of dataset
+  void printTitle(std::ostream& os) const final { os << GetTitle(); }
+  void printClassName(std::ostream& os) const final ;
+  void printArgs(std::ostream& os) const final;
+  /// Print value of the dataset, i.e. the sum of weights contained in the dataset
+  void printValue(std::ostream& os) const final { os << numEntries() << " entries" ; }
+  void printMultiline(std::ostream& os, Int_t content, Bool_t verbose, TString indent) const final;
```

Comment:
Attention: When you `final`ise previously virtual functions, external projects might not compile, any more. Maybe they didn't even want to override them because there was something to do, but some just override because they Ctrl+c,Ctrl+v the header of the base ...
I *guess* that it's OK here, but be prepared that people go around saying: "The RooFit guy broke my code."

At least it's worth a mini entry in the release notes.

Reply:
Okay, I didn't think anyone wants to overload this, but you made a good point with the copy-paste argument. I replaced the `final` keyword with `override` now.